version/version.go

```diff
@@ -17,19 +19,45 @@ import (
 
 //go:embed VERSION
 var baseVersion string
-var buildVersion string
+var buildNumber string
 
 func Version() string {
 	return strings.TrimSpace(baseVersion)
 }
 
-func BuildVersion() string {
-	if buildVersion == "" {
+func BuildNumber() string {
```

Comment:
The old name is a bit ambiguous. `BuildVersion` could be `v3.53.6992`, but it was just `6992`. `BuildNumber` does not have this ambiguity.